hello-world-notification-microservice/README.md

```diff
@@ -72,4 +72,43 @@ we will receive the following message as we have specified to transform the filt
   "source": {"id":"<DEVICE ID>"},
   "type": "c8y_Speed"
 }
-```
\ No newline at end of file
+```
+
+## Device creation and measurement sending
+
+The included [Python script](demo/measurements.py) has everything to get you started with the microservice and has example code for device creation, deletion and sending out measurements.
+To use the script you will need to provide authentication details, edit the script and provide values to the following fields:
+
+```python
+_tenant_id = ''
+_username = 'admin'
+_password = ''
+_platform_url = ''
+```
+
+To use, run the script with one of the options:
+
+### To create a device
+```console
+user@host:~$ python measurements.py device
+user@host:~$ Device id: <DEVICE_ID>
+```
+
```

Comment:
Would a snippet of the file showing the example.source.id line being set be nice here? Maybe continuing on to use the (made up) example actual device-id from above if we use that approach above

Reply:
We should definitley make it clear we are talking about the microservice properties file